packages/astro/test/astro-global.test.js

```diff
@@ -2,14 +2,15 @@ import assert from 'node:assert/strict';
 import { after, before, describe, it } from 'node:test';
 import * as cheerio from 'cheerio';
 import { loadFixture } from './test-utils.js';
+import testAdapter from './test-adapter.js';
 
 describe('Astro Global', () => {
 	let fixture;
 
 	before(async () => {
 		fixture = await loadFixture({
 			root: './fixtures/astro-global/',
-			site: 'https://mysite.dev/blog/',
+			site: 'https://mysite.dev/subsite/',
```

Comment:
A subpath to the site seems unnatural. It is possible it was not the intention of the orignal PR in the first place.
- https://github.com/withastro/astro/pull/3959.